### PR TITLE
fix: remove dev flag for cloudinary

### PR DESCRIPTION
**Pull request by palakgupta2712**
### What is the change?
Removal of dev feature flag for profile picture on members page.
Closes issue https://github.com/Real-Dev-Squad/website-members/issues/369

### Is it bug?
No 

### Before / After Change Screenshots

Before - 
Screenshot from 2022-05-08 19-33-26

After - 
Screenshot from 2022-05-08 19-33-08

### CODE REVIEW

File: src/pages/[id]/index.js [hunk 1]
```diff
@@ -74,7 +73,7 @@ export async function getServerSideProps(context) {
     }
     const { user } = await res.data;
     const getImageLink = (transformString) => {
-      return !!dev && user.picture
+      return user.picture
         ? getCloudinaryImgURL(user.picture.publicId, transformString)
         : getImgURL(user.username, 'img.png');
```
Comment (pallabez) [inline src/pages/[id]/index.js:78]: Can we remove this function call? We are planning on deprecating this.

Comment (palakgupta2712) [reply src/pages/[id]/index.js]: made the necessary changes,

**Review by pallabez — APPROVED** [review]
LGTM :+1:

**Review by rohan09-raj — APPROVED** [review]
Looks good to me 🔥 🔥

**Review by rohan09-raj — COMMENTED** [review]
@palakgupta2712 Can you please check and confirm if the feature flag is working for the skill tag and `Get Intro` button ??
image

**Comment by palakgupta2712** [comment]
@rohan09-raj 

## without feature flag
Screenshot from 2022-05-08 23-58-01


## with feature flag
Screenshot from 2022-05-08 23-57-42

**Review by ankushdharkar — APPROVED** [review]

**Pull request merged**